requirements.txt

```diff
@@ -13,5 +13,8 @@ torchvision>=0.16.0
 jax[cpu]
 flax
 
+# OpenVINO
+openvino
```

Comment:
Sorry I'm not super familiar, are these user-facing or development dependencies? Want to confirm this won't add another dep for users unless they opt in.

Reply:
@fchollet, can you please comment?
AFAIK, it used for testing preparation.

Reply:
`requirements.txt` is dev-facing not user facing. User facing deps are listed in pyproject.toml.

Reply:
If there is never a need to import openvino when testing another backend, we should create an openvino specific requirements file (like what we have for jax/torch/tf with cuda) and install that for the openvino CI run only.

Reply:
Sounds good, thanks

Reply:
@fchollet, @jeffcarp, I added custom requirements file for openvino.